src/decision/include/LidarDecision.h

```diff
@@ -63,21 +77,21 @@ class LidarObstacle {
      *
      * @return the angle of the Obstacle from the robot
      */
-    float getAvgAngle();
+    angle_t getAvgAngle();
 
     /**
      * Gets the minimum angle from of an object from the robot
      *
      * @ return the minimum angle of the obstacle from the robot
```

Comment:
Sorry, bad phrasing on my part I think, could you **remove** all spaces after `@`. ie. instead of `@ param` it should be `@param`